### PR TITLE
Fix nested edit bar

### PR DESCRIPTION
**Pull request by terapyon**
BUG REPORT: Some nested `section id="edit-bar"` tag in folder_contents page #2322 for Plone 5.1

### CODE REVIEW

**Comment by gforcada** [comment]
@terapyon thanks for the bug fix, but your commit unfortunately was based on a quite old version of CMFPlone. Would you mind rebasing your commit? Is it meant for 5.1 or 5.2? :thinking: you might have to change the branch you based your commit on.

**Comment by terapyon** [comment]
@gforcada Thank you for your comment. I want to fix on Plone 5.1.x.
Please let me know how to make a branch. Because I don't have a enough skill for github, I'm sorry.

**Comment by gforcada** [comment]
Oh, now I see, the commits were fine, just that when you created the pull request, github by default selected master branch as the target branch to merge yours, while you wanted to merge it against branch 5.1.x

Fortunately, the edit button allows one to change the target branch :slightly_smiling_face:

**Comment by jensens** [comment]
The tests that are broken here have no connection to the change. My PR #2342 has the exact same failure. Any idea what's going on here?

File: Products/CMFPlone/static/patterns/toolbar/src/toolbar.js [hunk 1]
```diff
@@ -390,7 +390,7 @@ define([
           url: $('body').attr('data-portal-url') + path + '/@@render-toolbar'
         }).done(function(data) {
           var $el = $(utils.parseBodyTag(data));
-          that.$el.replaceWith($el);
+          that.$el.parent.replaceWith($el);
```
Comment (thet) [inline Products/CMFPlone/static/patterns/toolbar/src/toolbar.js]: That broke toolbar reloading when browsing folder_contents.
That also shows the importance of having tests, which we apparently do not have here.

Comment (thet) [inline Products/CMFPlone/static/patterns/toolbar/src/toolbar.js]: It should read ```that.$el.parent().replaceWith($el);```

**Comment by thet** [comment]
@terapyon Maybe #2324 is something specific to a special integration project of yours? With my fixes in place the site is even more broken than before. see: https://github.com/plone/Products.CMFPlone/pull/2398

**Comment by thet** [comment]
@terapyon no, it is not specific to your problem. I can reproduce the problem. However, this fix needs a different approach.

**Pull request merged**